app/components/to-do-list.js

```diff
@@ -0,0 +1,45 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { v4 as uuidv4 } from 'uuid';
+
+export default class ToDoListComponent extends Component {
+
+    description = '';
```

Comment:
A suggestion here would be to make the format of these properties be similar to the structure that you are going to be storing in the toDoListItems.